cluster/pulumi/gha/src/config.ts

```diff
@@ -7,6 +7,11 @@ import { z } from 'zod';
 const GhaConfigSchema = z.object({
   gha: z.object({
     githubRepo: z.string(),
+    // these a Splice versions
+    runnerVersion: z.string(),
```

Comment:
Is there ever a good reason where runnerVersion and runnerHookVersion should be different?